policy/DecamMapper.paf

```diff
@@ -755,4 +755,12 @@ datasets: {
         persistable:   "Config"
         storage:       "ConfigStorage"
     }
+    singleFrameDriver_config: {
+        template:      "config/singleFrameDriver.py"
+        python:        "lsst.pipe.drivers.singleFrameDriver.SingleFrameDriverConfig"
+        persistable:      "Config"
+        storage:    "ConfigStorage"
+        tables:        raw
+        tables:        raw_visit
+    }
```

Comment:
I thought so conceptually, but somehow it gave this warning: 
`singleFrameDriver WARNING: Could not persist metadata for dataId={'date': '2013-09-01', 'filter': 'g', 'visit': 229341}: 'DecamMapper' object has no attribute 'map_singleFrameDriver_metadata`